dist/components/Checkbox/Checkbox.d.ts

```diff
@@ -7,6 +7,7 @@ interface CheckboxProps {
     formikKey?: string;
     tableDeselect?: boolean;
     'data-testid'?: string;
+    backgroundColor?: string;
```

Comment:
string to nie jest najlepszy pomysł proponuję:
```suggestion
    backgroundColor?: 'color-1' | 'color-1'...itd;
```